inc/git-updater/namespace.php

```diff
@@ -20,24 +20,20 @@ function on_load() : void {
 		return;
 	}
 
-	add_action( 'gu_get_remote_plugin', __NAMESPACE__ . '\\update_on_cron', 20, 1 ) ;
+	add_action( 'get_remote_repo_meta', __NAMESPACE__ . '\\update_on_get_remote_meta', 20, 2 ) ;
 }
 
 /**
- * Update necessary FAIR data on the Git Updater cron event.
+ * Update necessary FAIR data during the Git Updater get_remote_repo_meta().
  *
- * @todo Switch this to running on all updates, requires a change in GU.
- *
- * @param array $batches List of repositories to update.
+ * @param \stdClass $repo Repository to update.
+ * @param GitHub_API      Repository API object.
```

Comment:
Technically this parameter is one of the following

- \Fragen\Git_Updater\API\GitHub_API
- \Fragen\Git_Updater\API\Bitbucket_API
- \Fragen\Git_Updater\API\GitLab_API
- \Fragen\Git_Updater\API\Gitea_API